src/iam_users/list_iam_user_resources_cost.py

```diff
@@ -246,4 +271,4 @@ def lambda_handler(event, context):
     # Calling method to get cost of resources
     cost_of_resources(event, case_list, account_id)
 
-    return {"statusCode": 200, "body": json.dumps(case_list)}
+    return {"statusCode": 200, "body": json.dumps(case_list)}
```

Comment:
EOF missing

Reply:
Added in d0ff7d06072eb4c290cb31d2d612c49dd5c6e6ed